glances/Dockerfile

```
@@ -12,6 +12,7 @@ COPY requirements.txt /tmp/
 # hadolint ignore=DL3042
 RUN \
     apk add --no-cache --virtual .build-dependencies \
+        gcc=11.2.1_git20220219-r2 \
```

This is lower maintenance (changes way less often, as it is a meta package), and also provides gcc.

```suggestion
        build-base=0.5-r3 \
```